### PR TITLE
Add cases time.Time and nil to replaceInsertValueFromValArg function

### PR DESCRIPTION
**Pull request by TatsuNet**
Change to convert value when `time.Time` and `nil` are received.

### CODE REVIEW

File: sqlparser/sqlparser.go [hunk 1]
```diff
@@ -223,6 +224,27 @@ func (p *Parser) replaceInsertValueFromValArg(query *InsertQuery, colIndex int,
 				Val:  []byte(fmt.Sprintf("%d", arg)),
 			}
 		}
+	case time.Time:
```
Comment (goccy) [inline sqlparser/sqlparser.go]: May collect cases of `time.Time` and `*time.Time` ?

File: sqlparser/sqlparser.go [hunk 2]
```diff
@@ -223,6 +224,27 @@ func (p *Parser) replaceInsertValueFromValArg(query *InsertQuery, colIndex int,
...
+			}
+		}
+	case *time.Time:
+		query.ColumnValues[colIndex] = func() *vtparser.SQLVal {
+			return &vtparser.SQLVal{
+				Type: vtparser.StrVal,
+				Val:  []byte(arg.Format("2006-01-02 15:04:05")),
+			}
+		}
+	case nil:
```
Comment (goccy) [inline sqlparser/sqlparser.go]: I also concern about other pointer types like `*int` or `*uint` and so on..

File: sqlparser/sqlparser.go (hunk 1 above)
Comment (TatsuNet) [reply sqlparser/sqlparser.go]: https://github.com/knocknote/octillery/pull/17/commits/f87c53632ad01c8921970311c37cfc11c996734d

File: sqlparser/sqlparser.go (hunk 2 above)
Comment (TatsuNet) [reply sqlparser/sqlparser.go]: https://github.com/knocknote/octillery/pull/17/commits/4a79d668613579752f09f1ff1616b4b9b8800e90

File: sqlparser/sqlparser.go [hunk 3]
```diff
@@ -197,30 +202,186 @@ func (p *Parser) replaceInsertValueFromValArg(query *InsertQuery, colIndex int,
...
+			query.ColumnValues[colIndex] = createSQLNilTypeVal()
+		} else {
+			query.ColumnValues[colIndex] = createSQLIntTypeVal(*arg)
+		}
+	case bool:
+		val := convertBoolToInt8(arg)
+		query.ColumnValues[colIndex] = createSQLIntTypeVal(val)
+	case *bool:
+		if arg == nil {
+			query.ColumnValues[colIndex] = func() *vtparser.SQLVal {
```
Comment (goccy) [inline sqlparser/sqlparser.go]: This section can refactor by `createSQLNilTypeVal`

Comment (TatsuNet) [reply sqlparser/sqlparser.go]: https://github.com/knocknote/octillery/pull/17/commits/cf85cdc8ca29bed49254907192b1624246b5c5fd

**Comment by goccy** [comment]
LGTM !!!!

**Pull request merged**